sokol_app.h

```diff
@@ -11378,8 +11499,7 @@ SOKOL_API_IMPL const char* sapp_get_clipboard_string(void) {
     #elif defined(_SAPP_WIN32)
         return _sapp_win32_get_clipboard_string();
     #else
-        /* not implemented */
-        return _sapp.clipboard.buffer;
+        return _sa_sapp_x11_get_clipboard_string();
```

Comment:
...ditto here (`#elif defined(_SAPP_LINUX)` and the generic `#else` needs to remain as before.